src/CLI.ts

```diff
@@ -40,6 +40,13 @@ export default class CLI {
             alias: "h",
             type: Boolean,
             description: "Display this help page (and ignore all other options)"
+        },
+        {
+            name: "filter",
+            alias: "f",
+            type: String,
+            typeLabel: "[filename]",
```

Comment:
Could this be the `SuiteName.TestName` mentioned below, to replace the parentheses?

Reply:
Great point. Now that the previously required association between file name and class name has been removed, I've updated this usage hint accordingly (see review comment below this).